tests/Unit/PointwiseFunctions/Hydro/TestFunctions.py

```diff
@@ -36,7 +36,9 @@ def mass_weighted_coords_b(tilde_d, grid_coords, compute_coords):
     return tilde_d * compute_coords * np.heaviside(grid_coords * (-1.0), 0)
 
 
-# Functions testing the MassWeightedFluidItems
+# End functions for testing the MassWeightedFluidItems
+
+# Functions for testing ComovingMagneticField.cpp
```

Comment:
can you just drop the changes in this file? Someone else is splitting it into multiple, so it's best to avoid conflicts. In general, doing cleanups like this are greatly appreciated!